src/command_queue.rs

```diff
@@ -1240,19 +1255,20 @@ impl CommandQueue {
         Ok(get_command_queue_info(self.queue, CommandQueueInfo::CL_QUEUE_PROPERTIES)?.to_ulong())
     }
 
+    #[cfg(feature = "CL_VERSION_2_0")]
```

Comment:
I think that this is incorrect.  
The call to `get_command_queue_info` should be in every OpenCL ICD, it is not an "CL_VERSION_2_0" feature.  
Therefore, since `CL_QUEUE_PROPERTIES` is just an attribute of an OpenCL 2 device it should be accessible from *any* OpenCL ICD and **not** an "CL_VERSION_2_0" feature.

Perhaps it should just be commented as CL_VERSION_2_1 was on line 1247 below.